deployment/cloud-run/secure_api_server.py

```
@@ -32,6 +31,15 @@
 )
 logger = logging.getLogger(__name__)
 
+# Security configuration from environment variables
+ADMIN_API_KEY = os.environ.get("ADMIN_API_KEY")
+if not ADMIN_API_KEY:
+    raise ValueError("ADMIN_API_KEY environment variable must be set")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)